zerver/openapi/zulip.yaml

```diff
@@ -8261,6 +8266,89 @@ paths:
       responses:
         "200":
           $ref: "#/components/responses/SimpleSuccess"
+  /user_topics:
+    post:
+      operationId: update-user-topic
+      summary: Update visibility policy of a topic
```

Comment:
Same naming comment here and in the sidebar index -- "Update personal preferences for a topic" is a better summary; in the longer-form description I'd extend the sentence to say "such as the topic's visibility policy, which is used to implement [muted topics](/help/mute-a-topic) and related features."

Reply:
Updated the docs.